### ISSUE TITLE
[Bug]: mometric checkNewAccountSize bug

### ISSUE THREAD

**Issue by fengttt** [matrixorigin/matrixone#12458]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 1.0-dev, mo cloud bd9f86e8b
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

errro during checkNewAccountSize, column name does not exist, created_time
```
Wraps: (2)
  -- stack trace:
  | github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlResultSet).columnName2Index
  | 	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/resultset.go:298
  | github.com/matrixorigin/matrixone/pkg/frontend.(*internalExecResult).StringValueByName
  | 	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/internal_executor.go:138
  | github.com/matrixorigin/matrixone/pkg/util/metric/mometric.checkNewAccountSize
  | 	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/mometric/cron_task.go:260
  | runtime.goexit
```

**Comment by xzxiong** [comment]
check branch main fixed
double checking. If alright, will pick to 1.0-dev branch

**Comment by xzxiong** [comment]
wait https://github.com/matrixorigin/matrixone/pull/12468 merged first, the pick #12473 into 1.0-dev

**Comment by xzxiong** [comment]
fixed, after #12473 on main and #12500 on 1.0-dev

**Comment by xzxiong** [comment]
@fengttt ptal
